### ISSUE TITLE
Request: Stream of data response

### ISSUE THREAD

**Issue by richfisher** [azlyth/react-native-ssh#2]
Handle the response goes on and on

Example: ping, top

**Comment by azlyth** [comment]
Yes, right. Currently, the library will only return when the command
finishes. Meaning streams won't work. I mean to look into this in the
future.
